packages/material-ui/src/Menu/Menu.js

```diff
@@ -116,8 +171,27 @@ const Menu = React.forwardRef(function Menu(props, ref) {
   });
 
   const items = React.Children.map(children, (child, index) => {
+    if (!React.isValidElement(child)) {
```

Comment:
Not sure if it matters, but they weren't testing for this before. Just marking it for discussion 🤷

Reply:
They actually are testing for this already. https://github.com/mui-org/material-ui/blob/b17e6856c33e96e1b6c5c5dfac327127cf4be705/packages/material-ui/src/Menu/Menu.js#L94

Reply:
completely missed that, good call